Bottom: 8d2f75897b59b2230d826c1aa131f191ac2a54cb
Top:    c4d4709d52e2de492239ded2a4b74e4ebde0db7a
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-12-18 15:05:17 -0800

Refresh of dsocket

---

diff --git a/docs/rsocket b/docs/rsocket
index a660208..4192450 100644
--- a/docs/rsocket
+++ b/docs/rsocket
@@ -219,4 +219,5 @@ the send.
 A service thread is used to process messages received on the UDP socket.  This
 thread updates the rsocket lookup tables with the remote QPN and path record
 data.  The service thread forwards data received on the UDP socket to an
-rsocket QP.
\ No newline at end of file
+rsocket QP.  After the remote QPN and path records have been resolved, datagram
+communication between two nodes are done over the UD QP.
\ No newline at end of file
